app/views/employees/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Name:</strong>
+  <%= @employee.name %>
+</p>
+
+<p>
+  <strong>Email:</strong>
+  <%= @employee.email %>
+</p>
+
+<p>
+  <strong>Private number:</strong>
+  <%= @employee.private_number %>
+</p>
+
+<p>
+  <strong>User:</strong>
+  <%= @employee.user_id %>
+</p>
+
+<%= link_to 'Edit', edit_employee_path(@employee) %> |
+<%= link_to 'Back', employees_path %>
```

Comment:
I recommend that you style this part of code where the information of a single employee is displayed.

Reply:
@YilKanda  The styles for all this scaffold elements are going to be in a different issued.